print/components/story/StoryDay.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="tw-mb-4">
+    <div class="tw-text-2xl tw-mb-2 tw-break-after-avoid">
+      <h1>{{ $tc('entity.day.name') }} {{ day.number }} ({{ dateLong(day.start) }})</h1>
+    </div>
+
+    <template v-if="entriesWithStory.length">
+      <template v-for="{ scheduleEntry, storyChapters } in entriesWithStory">
+        <div v-for="chapter in storyChapters" :key="chapter._meta.uri" class="tw-mb-3">
+          <h4 class="tw-text-lg tw-font-bold">
+            <span class="d-inline-flex align-center">
+              <span>{{ scheduleEntry.number }}</span>
+              <category-label
+                :category="scheduleEntry.activity().category()"
+                class="tw-ml-1"
+              />
+            </span>
+
+            <span>{{ scheduleEntry.activity().title }}</span>
+            <template v-if="chapter.instanceName">
+              - {{ chapter.instanceName }}
+            </template>
+          </h4>
+
+          <rich-text :rich-text="chapter.text" />
+        </div>
+      </template>
+    </template>
+    <p v-else>
+      {{ $tc('story.storyDay.noStory') }}
+    </p>
+
+    <hr />
+  </div>
+</template>
+
+<script>
+import CategoryLabel from '../generic/CategoryLabel.vue'
+import { dateLong } from '@/../common/helpers/dateHelperUTCFormatted.js'
+
+export default {
+  components: { CategoryLabel },
+  props: {
+    day: { type: Object, required: true },
+    index: { type: Number, required: true },
+    periodStoryChapters: { type: Array, required: true },
+  },
+  data() {
+    return {}
+  },
+  computed: {
+    // returns scheduleEntries of current day without the need for an additional API call
+    scheduleEntries() {
+      return this.day
+        .period()
+        .scheduleEntries()
+        .items.filter((scheduleEntry) => {
+          return scheduleEntry.day()._meta.self === this.day._meta.self
+        })
+    },
+
+    entries() {
+      return this.scheduleEntries.map((scheduleEntry) => ({
+        scheduleEntry,
+        storyChapters: this.periodStoryChapters.filter(
```

Comment:
Ich würde hier leere story chapters rausfiltern. Im web UI machts Sinn diese zu behalten, da sie potenziell bearbeitet werden müssen. Aber im PDF ist das nur unnötiger Noise der den Lesefluss stört

Reply:
Agree, will implement

Reply:
Implemented with https://github.com/ecamp/ecamp3/pull/2828/commits/7d54dd2a087acb398b1e4458c16fb4bd0182a7ea
PDF in PR description is updated